Philips.CodeAnalysis.Test/Verifiers/DiagnosticVerifier.Helper.cs

```diff
@@ -227,9 +227,9 @@ protected Document CreateDocument(string source)
 			return CreateProject(new[] { source }).Documents.First();
 		}
 
-		protected virtual MetadataReference[] GetMetadataReferences()
```

Comment:
Because the only ones left are a single MetadataReference, we could remove support for collections? Though maybe someone would need it later.

Reply:
I'll leave it as-is just in case